### PR TITLE
Users view

### PR DESCRIPTION
**Pull request by HelenaEksler**

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu Ready

File: db-update.example.sh [hunk 1]
```diff
@@ -23,4 +23,5 @@ drush sql-query "UPDATE field_data_field_github_access_token SET field_github_ac
 
 echo "Disable logs module"
 drush dis logs_http -y
+drush en views_ui -y
```
Comment (amitaibu) [inline db-update.example.sh]: remove

File: upgrade_scripts/users-view.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```
Comment (amitaibu) [inline upgrade_scripts/users-view.sh]: lets prefix the file name by issue number

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**